taxonomies/brands.txt

```diff
@@ -0,0 +1,12 @@
+# All brands in the brands taxonomy are language less and use the xx: prefix
+# Brands are not translated, we record what is written on the package, and we display it back as-is
+
+xx: Aldi
```

Comment:
so we could add a property like `wikidata:en:Q125054` below ?

Reply:
Sounds ok as it is a standard element.

Reply:
yes, because https://www.wikidata.org/wiki/Q125054 is for both Aldi

we would need to be careful with brands that use the same name (e.g. "Pacific" for alcohol and for other stuff: https://world.openfoodfacts.org/brand/pacific ).